Equilobe.DailyReport.SL/DataService.cs

```diff
@@ -482,7 +482,70 @@ public void SaveIpnLogError(long id, string error)
             }
         }
 
+        public List<SourceControlOptions> GetAllReposSourceControlOptions(long instanceId)
+        {
+            var sourceControlOptions = new List<SourceControlOptions>();
+
+            using (var db = new ReportsDb())
+            {
+                var policyStrings = GetInstancePolicyStrings(instanceId);
+
+                if (!policyStrings.Any())
+                    return sourceControlOptions;
+
+                foreach (var policyString in policyStrings)
+                {
+                    var advancedSettings = new AdvancedReportSettings();
+
+                    Deserialization.XmlDeserialize<AdvancedReportSettings>(policyString)
+                        .CopyPropertiesOnObjects(advancedSettings);
+
+                    sourceControlOptions.Add(advancedSettings.SourceControlOptions);
+                }
+            }
+
+            return sourceControlOptions;
+        }
+
+        public List<User> GetInstanceUsers(long instanceId)
```

Comment:
Hmm. This class User is a very old one, from the time we stored the policies in xml files. So this is more like a user mapping actually, where we mapped jira users with source control users. So maybe not change the class name now, but maybe this method should better be called something like GetUserMappings or something similar.